documentation/cubic-pos-article.md

```diff
@@ -0,0 +1,155 @@
+
+Namada's PoS module is similar to and influenced by the PoS system of the Cosmos Hub. Like many other blockchains in the Cosmos ecosystem, Namada employs Tendermint as its PoS consensus engine. Tendermint is a byzantine fault-tolerant (BFT) system that provides deterministic state machine replication as long as at least $2/3$ of the consensus voting power agrees on the state of the machine. Voting power in the consensus mechanism is held by nodes called validators, and the voting power is proportional to the amount of staked tokens held by the validators.
+
+- Anything else?
+
+This article is divided into three main sections. First, we describe the mechanics and configuration of the epoched PoS system. Then we describe Namada's novel approach to slashing for validator misbehavior. Finally, we describe the inflation and rewards system within the PoS module.
+
+
+# Epoched Proof of Stake
+
+The native staking token in Namada is identified as *NAM*. We call the smallest denomination of this token *namnam*, where 1 million *namnam* is equal to 1 *NAM*.
+
+Namada's epoched PoS model is influenced by that of [Cosmos](https://github.com/cosmos/cosmos-sdk/blob/019444ae4328beaca32f2f8416ee5edbac2ef30b/docs/architecture/adr-039-epoched-staking.md#pipelining-the-epochs). Sets of blocks are batched into epochs, which are periods of time that have some fixed configuration of the PoS system. For example, the set of validators who participate in the consensus protocol and their associated consensus voting powers can only change at the boundary of two epochs and are constant throughout the duration of one.
+
+Namada employs epoched staking for two main reasons:
+- Light clients become more efficient because there are fewer state changes (particularly validator set changes) to store, resulting in fewer headers needed to verify state
+	- TODO: maybe need to reword
+	- Q: does this light client efficiency apply only to IBC or also for Eth?
+- Eventually, Namada (and the larger Anoma ecosystem) will rely on the Ferveo DKG protocol to encrypt transactions in the mempool. Ferveo must know the validator set for a near-future epoch in order to have sufficient time to compute the DKG in advance of the given epoch.
+
+Thus, changes to the state of the PoS system are enqueued to take effect at some epoch in the future, the lengths of which we refer to as *offsets* relative to the epoch in which the transaction is submitted. Most state changes, including validator set updates, take effect at the *pipeline offset* - 2 epochs into the future. The other important offset in the model is the *unbonding offset* of 21 epochs, primarily the length of time before an unbond becomes withdrawable.
```

Comment:
wherever we list values, we should make it clear that they are parameters - e.g. "unbonding offset (by default, 21 epochs)"